.github/workflows/build.yml

```diff
@@ -65,13 +64,14 @@ jobs:
           if-no-files-found: error
           retention-days: 7
       - name: Upload Package
+        if: matrix.target != 'windows'
```

Comment:
why don't you upload package for windows? Add a comment for the specific reason please.